src/studio/studio.c

```diff
@@ -2334,20 +2334,16 @@ static void blitCursor(Studio* studio)
 
         tic_point hot = {0};
 
-        if(tic->ram->vram.vars.cursor.system)
+        bank = &getConfig(studio)->cart->bank0;
```

Comment:
You removed using of `vram.vars.cursor.system` flag which user could use to switch between system/custom cursors